src/app/App.tsx

```diff
@@ -4,13 +4,12 @@ import { hot } from 'react-hot-loader';
 import { Persistor } from 'redux-persist';
 
 import { AppRoutes } from './AppRoutes';
-import { Login, NewUser } from './screens';
 import { isLoggedIn, isNewUser } from './services';
 import { useDispatch, useSelector } from './store';
 import { setLoggedIn, setNewUser } from './store/auth';
 
 const App = ({ persistor }: { persistor: Persistor }) => {
-  const { loggedIn, newUser } = useSelector((state) => state.auth);
+  const { loggedIn } = useSelector((state) => state.auth);
```

Comment:
It's more efficient to select a variable in the state directly, to prevent re-renders when something in state.auth changes.

```suggestion
  const loggedIn = useSelector((state) => state.auth.loggedIn);
```